docs/getting-started.md

```diff
@@ -0,0 +1,127 @@
+# Getting started with Kalliope
+
+Kalliope needs two files to works, a `settings.yml` and a `brain.yml`.
+As files are written on YAML syntax, we strongly recommend you to use an editor(IDE) like [VS Code](https://code.visualstudio.com/) or [Atom](https://atom.io/).
+
+If you are using kalliope from a Rpi, the idea would be to configure your assistant from your main computer with an IDE and then push your config folder into your Rpi.
+
+When you start kalliope using the CLI (`kalliope start`), the program will try to load your `settings.yml` and `brain.yml` in the following order:
+
+- From your current folder, E.g `/home/pi/my_kalliope/`
+- From `/etc/kalliope/`
+- From the default `settings.yml` and `brain.yml` which are located in the root of the Kalliope project tree.
+
+This is a common tree of a Kalliope configuration folder:
+```
+kalliope_config/
+├── brains
+│   └── included_brain.yml
+├── brain.yml
+├── files
+│   └── kalliope-EN-13samples.pmdl
+└── settings.yml
+```
+
+We made starter kits that only needs to be cloned, placed into the Rpi and launched. You'll find the whole list of available start kits on the [Kalliope's website](https://kalliope-project.github.io/starter_kit.html).
+Those repositories provide you a structure to start playing and learning basics of Kalliope.
+Download the starter kit of your choice and open the folder with your IDE.
+
+All files are expressed in YAML format (see [YAML Syntax](https://learnxinyminutes.com/docs/yaml/)) and has a minimum of syntax, which intentionally tries to not be a programming language or script,
+but rather a model of a configuration or a process.
+
+Let's open the main brain file of the English starter kit. You'll see there are some included sub brains file.
+```yaml
+- includes:
+    - brains/say.yml
+```
+
+If you open the `say.yml` file from the brains folder, you'll see a basic **synapse** that uses the [neuron](brain/brain.md#neurons) "[Say](brain/neurons/say)" and make Kalliope speaks out loud "Hello sir" when you say "hello".
+```yaml
+- name: "say-hello-en"
+  signals:
+    - order: "Hello"
+  neurons:
+    - say:
+        message: "Hello sir"
+```
+
+Let's break this down in sections so we can understand how the file is built and what each part means.
+
+Items that begin with a ```-``` are considered as list items. Items have the format of ```key: value``` where value can be a simple string or a sequence of other items.
+
+At the top level we have a "name" tag. This is the **unique identifier** of the synapse. It must be an unique word with the only accepted values : alphanumerics and dash. ([a - zA - Z0 - 9\-])
+```yaml
+- name: "Say-hello"
+```
+
+The first part, called **signals** is a list of input actions.
+You can add as many signal as you want in the "signals" section. If one of them is triggered, te neuron list will be executed.
+```yaml
+signals:
+  - order: "say-hello"
+```
+
+In the following example, we use just one signal, an "order", but it an can be:
```

Comment:
maybe incluce link to "core signal" ?

Reply:
The core signal is not a page, it's a menu, I cannot add a link here